infrastructure/staging/ec2_host.tf

```diff
@@ -0,0 +1,67 @@
+resource "aws_instance" "access_host" {
+  ami           = data.aws_ami.amazon-linux-2.id
+  instance_type = "t2.micro"
+  key_name      = "wellcomedigitalworkflow"
```

Comment:
You might want to to turn this into a `var`.

Reply:
instance_type and key_name are variables now, defaulting to those values.